raiden-network/light-client
# Feature/dapp add wallet connect

**Thank you for submitting this pull request :)**

Fixes #2599

**Short description**
Ugly (for the moment) but working.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Provide an RPC endpoint for WalletConnect in the `public/config.json` (see the example config file)
2. Disable MetaMask in your browser
3. Serve the dApp locally (maybe in `--mode production` due to current test env transport issues)
4. When click on connect, a QR code should show up
5. Scan it with a mobile wallet up (don't use MetaMask it doesn't work, I recommend ImToken)
6. Make sure you can connect, and can sign the subkey request by the SDK
